presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -325,6 +328,13 @@ public Void visitMarkDistinct(MarkDistinctNode node, Void context)
             return node.getSource().accept(this, context);
         }
 
+        @Override
+        public Void visitNativeExecution(NativeExecutionNode node, Void context)
+        {
+            printNode(node, "NativeExecution", NODE_COLORS.get(NodeType.NATIVE_EXECUTION));
+            return null;
```

Comment:
should we visit the node.subPlan to get the plan that it's wrapping?

Reply:
@rschlussel We should do that, but in a follow-up. Currently, we are flooding the logs with long stack traces making it hard to read these logs.